tests/testthat/test-build-reference-index.R

```diff
@@ -18,7 +18,12 @@ test_that("warns if missing topics", {
   )
   meta <- list(reference = ref)
   pkg <- as_pkgdown(test_path("assets/reference"), override = meta)
+  old_ci <- Sys.getenv("CI")
```

Comment:
Can you please use `withr::local_envvar()` here?

Reply:
Done in 024ac28